MapleServer2/Packets/SkillUsePacket.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Maple2Storage.Types;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Types;
+
+namespace MapleServer2.Packets
+{
+    public static class SkillUsePacket
+    {
+        public static Packet SkillUse(int value, long count, int skillId, short skillLevel, CoordF coords)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.SKILL_USE);
+            pWriter.WriteLong(count);
+            pWriter.WriteInt(value);    // Unknown
+            pWriter.WriteInt(skillId);
+            pWriter.WriteShort(skillLevel);
+            pWriter.WriteByte();
+            pWriter.Write(coords);
+            pWriter.WriteLong();
+            pWriter.WriteLong();
+            pWriter.WriteInt();
+            pWriter.WriteShort();
+            pWriter.WriteInt();
+            pWriter.WriteByte();
+            pWriter.WriteByte();
+            return pWriter;
+        }
+
+        public static Packet SkillDamage(IFieldObject<Player> player, long skillCount, int someValue, int skillId, short skillLevel, CoordF coords, int objectId)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.SKILL_DAMAGE);
```

Comment:
Can this live in another file since it's a different opcode?

Reply:
Yeah, just had it there because it was a lit fast for testing purposes

Reply:
You can delete it from this file now that you committed it to another file